server/src/hexalot.ts

```diff
@@ -62,6 +62,29 @@ function* genSpiral(radius: number): IterableIterator<Vector> {
     }
 }
 
+export function overlap(h1: Hexalot, h2: Hexalot, direction: Direction): boolean {
```

Comment:
Will probably be replaced by the same logic as you wrote client-side

Reply:
In a subsequent PR?

Reply:
Yep